<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>metadataQuery</class>
 <widget class="QDialog" name="metadataQuery">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>751</width>
    <height>309</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>File information</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>true</bool>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="7" column="0">
    <layout class="QVBoxLayout" name="verticalLayout" stretch="0">
     <item>
      <widget class="QDialogButtonBox" name="bottomButtonsBox">
       <property name="sizePolicy">
        <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="standardButtons">
        <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
       </property>
       <property name="centerButtons">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <layout class="QHBoxLayout" name="metadataLayout">
     <property name="topMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QLabel" name="fileNameLabel">
       <property name="text">
        <string>Name of the file:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="fileNameEdit">
       <property name="enabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="3" column="0">
    <layout class="QHBoxLayout" name="ownerLayout">
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QLabel" name="ownerLabel">
       <property name="text">
        <string>Owner of file (name or SHA1 if CA operator):</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="ownerEdit"/>
     </item>
    </layout>
   </item>
   <item row="6" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="5" column="0">
    <layout class="QHBoxLayout" name="licenseLayout">
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QLabel" name="licenseLabel">
       <property name="text">
        <string>License of use:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="licenseComboBox">
       <property name="editable">
        <bool>true</bool>
       </property>
       <item>
        <property name="text">
         <string>Public domain</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>GPL</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>LGPL</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>BSD</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>CC-BY</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>CC-BY-SA</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>CC-BY-ND</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>CC-BY-NC</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>CC-BY-NC-SA</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>CC-BY-NC-ND</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
   <item row="4" column="0">
    <layout class="QHBoxLayout" name="mimetypeLayout">
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QLabel" name="mimetypeLabel">
       <property name="text">
        <string>Mime-type:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="mimetypeEdit"/>
     </item>
    </layout>
   </item>
   <item row="2" column="0">
    <layout class="QHBoxLayout" name="descriptionLayout">
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QLabel" name="descriptionLabel">
       <property name="text">
        <string>Description of file:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="descriptionEdit"/>
     </item>
    </layout>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="topDescriptionLabel">
     <property name="font">
      <font>
       <pointsize>13</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Enter information regarding file before publish:</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>fileNameEdit</tabstop>
  <tabstop>bottomButtonsBox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>bottomButtonsBox</sender>
   <signal>rejected()</signal>
   <receiver>metadataQuery</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>411</x>
     <y>653</y>
    </hint>
    <hint type="destinationlabel">
     <x>411</x>
     <y>335</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
